pkg/orchestrator/assetscanestimationwatcher/config.go

```diff
@@ -23,9 +23,10 @@ import (
 )
 
 const (
-	DefaultPollInterval     = 5 * time.Second
-	DefaultReconcileTimeout = 15 * time.Second
-	DefaultAbortTimeout     = 1 * time.Minute
+	DefaultPollInterval                  = 5 * time.Second
+	DefaultReconcileTimeout              = 15 * time.Second
+	DefaultAbortTimeout                  = 1 * time.Minute
+	DefaultAssetScanEstimationTTLSeconds = 7200 // 2 hours
```

Comment:
```suggestion
	DefaultAssetScanEstimationTTLSeconds = 7200 * time.Second // 2 hours
```

Reply:
Ahh, nevermind. It'd require a bit more change.

Reply:
could have been `2 * time.Hour` but nevermind...